### PR TITLE
FCBH-3319 auth for keys on fileset bulk requests

### PR DESCRIPTION
**Pull request by JramR7**
# Description

* THIS PR IS ONLY FOR AUTH of the key, the return of the desired bulk response will be addressed on another pr
* Create showBatch to return bulk request
* Create blockedByBulkAccessControl to check fileset permissions and if. the current key allows the bulk request

## Issue Link

[FCBH-3319](https://fullstacklabs.atlassian.net/browse/FCBH-3319)

## How Do I QA This

* Set env variable KEYS_FILESET_BULK_ACCESS={a key with filesets access in your local, or a group separated by commas}
* {basePath}/bibles/filesets/ENGESVP2DV/batch – should return 404 if key does not have permissions, return a fileset if it does

**Note:** To run the test suite refer to the Readme.md

### CODE REVIEW

File: app/Http/Controllers/Bible/BibleFileSetsController.php [hunk 1]
```diff
@@ -222,6 +222,76 @@ function () use ($fileset_id, $book_id, $chapter_id, $verse_start, $verse_end) {
         return $this->reply($fileset_chapters, [], $transaction_id ?? '');
     }
 
+    /**
+     *
+     * @OA\Get(
+     *     path="bibles/filesets/{fileset_id}/batch",
```
Comment (bradflood) [inline app/Http/Controllers/Bible/BibleFileSetsController.php:228]: please rename batch to bulk in all locations

File: app/Http/Controllers/Bible/BibleFileSetsController.php [hunk 2]
```diff
@@ -222,6 +222,76 @@ function () use ($fileset_id, $book_id, $chapter_id, $verse_start, $verse_end) {
...
+                    return $this->setStatusCode(404)->replyWithError(
+                        trans('api.bible_fileset_errors_404')
+                    );
+                }
+
+                $bulk_access_blocked = $this->blockedByBulkAccessControl($fileset);
+                if ($bulk_access_blocked) {
+                    return $bulk_access_blocked;
+                }
+
```
Comment (bradflood) [inline app/Http/Controllers/Bible/BibleFileSetsController.php:284]: do asset_id and bible need to be set? It doesn't appear they are used

**Review by bradflood — CHANGES_REQUESTED** [review]
see comments below

File: app/Http/Controllers/Bible/BibleFileSetsController.php (hunk 2 above)
Comment (JramR7) [reply app/Http/Controllers/Bible/BibleFileSetsController.php:284]: That's unused. code, that just code copied from other functions, the content returned is just a fake response, until the next story

**Review by bradflood — APPROVED** [review]
LGTM. Thanks Jose

**Pull request merged**